.env.example

```
@@ -7,4 +7,6 @@ POSTGRES_PASSWORD=super_secret_root_password
 APP_RUNTIME_PASSWORD=runtime_password
 APP_WORKER_PASSWORD=worker_password
 APP_ADMIN_PASSWORD=admin_password
+# Passwords with symbols @, :, /, ?, #, %, [, and ] need to be percent encoded for connection strings
+APP_ADMIN_PASSWORD_URLENC=admin_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_ADMIN_PASSWORD_URLENC key should go before the APP_RUNTIME_PASSWORD key

(UnorderedKey)